index.html

```diff
@@ -527,11 +584,27 @@ <h4 class="modal-title">Editing {{tunit_editor_target_label}}</h4>
 
     <!-- Phylotree -->
     <script src="http://d3js.org/d3.v3.min.js"></script>
-    <script src="http://veg.github.io/phylotree.js/phylotree.js"></script>
+    <script src="./lib/phylotree.js/phylotree.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js" charset="utf-8"></script>
 
     <!-- Our source code -->
     <script src="./js/curation-tool.js"></script>
+
+    <!-- Activate window-specific features -->
+    <script type="text/javascript"><!--
+        $(window).on('beforeunload', function (e) {
+            // If someone tries to navigate away from the window while the
+            // PHYX has been modified, ask users to confirm before leaving.
+
+            // Confirmation message to display to the user. Note that modern
+            // browsers do not display this message, but provide a generic
+            // "content has changed" dialog instead.
+            var confirmationMessage = 'This PHYX file has been modified! Are you sure you want to close the Curation Tool?';
```

Comment:
Tell the user what would happen if they went ahead. E.g., "_Your modifications will be lost if you close the Curation Tool. Please confirm to close, and cancel otherwise._"